core/clipboard/block_paster.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {BlockSvg} from '../block_svg.js';
+import {CopyData} from '../interfaces/i_copyable';
+import {IPaster} from '../interfaces/i_paster.js';
+import {State, append} from '../serialization/blocks';
+import {Coordinate} from '../utils/coordinate.js';
+import {WorkspaceSvg} from '../workspace_svg.js';
+
+export class BlockPaster implements IPaster<BlockCopyData, BlockSvg> {
+  paste(
+    copyData: BlockCopyData,
+    workspace: WorkspaceSvg,
```

Comment:
Does the existing copy/paste system work with headless workspaces? If no, we can decide that working in headless is out of scope, and then this code gets to be simple. If yes, we have to decide whether to keep that or deliberately break it.

Reply:
Current code only works in rendered workspaces! `Block`s are not `ICopyable` only `BlockSvg`s are =) Same for workspace comments.